core/tests/protractor/editorAndPlayer.js

```diff
@@ -159,6 +208,22 @@ describe('Full exploration editor', function() {
         forms.toRichText('Parent Exploration Content'));
       explorationPlayerPage.submitAnswer('MultipleChoiceInput', 'Correct');
       browser.waitForAngular();
+
+      browser.get('/search/find?q=');
+      element.all(by.css(
+        '.protractor-test-collection-summary-tile-title')).first().click();
+      element.all(by.css(
+        '.protractor-test-collection-exploration')).first().click();
+      explorationPlayerPage.submitAnswer('MultipleChoiceInput', 'Incorrect');
+      general.waitForSystem();
+      explorationPlayerPage.clickConfirmRedirectionButton();
+      browser.waitForAngular();
+      browser.getCurrentUrl().then(function(url) {
```

Comment:
Add one-line comment above this to explain what it's checking

Reply:
Done